packages/hfs/src/babelRemoveImportsGetMeta/index.ts

```diff
@@ -0,0 +1,31 @@
+import { isModule } from '@babel/helper-module-transforms';
+import normalizeAndLoadModuleMetadata from './normalize-and-load-metadata';
+import rewriteLiveReferences from '@babel/helper-module-transforms/lib/rewrite-live-references';
+
+export default (stats) => (babel) => ({
```

Comment:
this function change state as side effect.
if you want to change state, return an immutable state.